wiki/markdown/usage/service.md

```diff
@@ -204,9 +204,12 @@ https://localhost:8080/eventrepository/events/?meta.type=EiffelArtifactCreatedEv
                 }]
 ```
 
-#### Lookups are provided with two options:
+#### Lookups are provided with four options:
 
 | Options              | Default Value | Description                                                                                                                                                                                                                                                                                                                                                                                                                            |
 |----------------------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | failIfMultipleFound: | False         | If value is set to false, REMReM generates event if multiple event ids found or no event id fetched from the event repository configured in generate and adds them in links section of the target field. If value is set to true, if only one event is found then REMReM generates the event successfully and if multiple event ids are found then REMReM fails to generate any event and displays relevant error message to the user. |
-| failIfNoneFound:     | False         | If value is set to true, no event ids fetched from event repository , REMReM fails to generate any event and displays relevant error message to the user. If value is set to false,REMReM generates event if no event found or atleast one event fetched from the event repository configured in configuration file.                                                                                                                   |
\ No newline at end of file
+| failIfNoneFound:     | False         | If value is set to true, no event ids fetched from event repository , REMReM fails to generate any event and displays relevant error message to the user. If value is set to false,REMReM generates event if no event found or atleast one event fetched from the event repository configured in configuration file.|                                                                                                 
+| shallow:             | True          | This parameter is to control the behavior of ER towards external ERs,default is true which states the ER queries can be connected to internal ER.else the ER queries can be connected to external ERs.|
+| pageSize:            | 1             | This property is similar to limit in mongoDB. You can specify a value greater than 0, and can say how many events you want default 1.
```

Comment:
The parameter should be renamed to 'limit'. The description could be:
The number of events returned, through any lookup definition given, is limited to this number.

Reply:
Done